x/programs/rust/examples/timelock/src/lib.rs

```diff
@@ -0,0 +1,313 @@
+use borsh::{BorshDeserialize, BorshSerialize};
+#[cfg(not(feature = "bindings"))]
+use wasmlanche_sdk::Context;
```

Comment:
🤔 What if we just made this an enum?

Not for this PR, but just a thought

Reply:
Can you please explain for what purpose ?